lib/model/edge.js

```diff
@@ -24,6 +24,52 @@ var Port = require('./port');
 var Edge = createClass({
   extend: GraphModelBase,
 
+  instance: {
+    /**
+     * Get the start position of an edge in the context of `graph`
+     *
+     * Returns null if the edge does not start in this graph.
+     * (Maybe it starts in a sub- or parent container?)
+     * 
+     * @method getStartPositionIn
+     * @param  {Graph} graph the graph to look for the node in. 
+     *                       Result is in its coordinate space.
+     * @return {Object|null} object with x and y
+     */
+    getStartPositionIn: function(graph) {
+      var port = this.get('from');
+      
+      return this._getPortPositionIn(port, graph);
+    },
+
+    /**
+     * Get the end position of an edge in the context of `graph`
+     *
+     * Returns null if the edge does not start in this graph.
+     * (Maybe it starts in a sub- or parent container?)
+     * 
+     * @method getEndPositionIn
+     * @param  {Graph} graph the graph to look for the node in. 
+     *                       Result is in its coordinate space.
+     * @return {Object|null} object with x and y
+     */
+    getEndPositionIn: function(graph) {
+      var port = this.get('to');
+      
+      return this._getPortPositionIn(port, graph);
+    },
+
+    _getPortPositionIn: function(port, graph) {
+      var nodeId = port.get('ownerVertexId');
+      var node = graph.getVertexById(nodeId);
+      if (node === null) {
+        return null;
+      }
+
+      return node.getPortPositionInContext(port.get('position'), true);
```

Comment:
It seems a little weird that you have to get the port's vertex and then get the port's position against the vertex. Shouldn't the port be able to handle this without your needing to grab the vertex? I've looked at how the method works, but it seems like that method should be a member of port rather than of vertex.